gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/base/mixin/DataTargetMixin.java

```diff
@@ -34,15 +35,28 @@
  */
 public class DataTargetMixin<T extends UIObject & HasDataTarget> extends AbstractMixin implements HasDataTarget {
 
+    /**
+     * Ensures the given element has a non-empty id.
+     *
+     * @param element The element being examined.
+     * @see Document#createUniqueId()
+     */
+    public static void ensureId(final Element element) {
```

Comment:
Ya, I agree.  I couldn't find a helper class to move it to.  Perhaps you could make it private...?  I'm not at work today.